contracts/stellar-interchain-token-service/src/contract.rs

```diff
@@ -813,21 +819,14 @@ impl InterchainTokenService {
             unregistered_token_id.clone(),
             token_metadata,
         );
-        let interchain_token_client = InterchainTokenClient::new(env, &token_address);
 
-        let token_manager = Self::deploy_token_manager(
+        let _ = Self::deploy_token_manager(
```

Comment:
```suggestion
        Self::deploy_token_manager(
```
does this function need to return anything at all anymore?